### PR TITLE
Allow multiple topic definitions.

### PR DESCRIPTION
**Pull request by honzas**
Allow multiple topic definitions.

We allowed an array in definition of topics - basically its a refactor of current method for creating a topic, only we apply it for all items in topic array.

All examples and motivation is in updated readme, examples and tests.

Everything backward compatible.

### CODE REVIEW

**Comment by anthonykaras** [comment]
Awesome work! This is exactly what we need. 
Any idea when this will be merged?

**Comment by bulkan** [comment]
@honzas can you please resolve the conflicts now ?

**Comment by honzas** [comment]
@bulkan Done

File: README.md [hunk 1]
```diff
@@ -67,6 +67,67 @@ functions:
...
+
+    topics:
+      critical:
+        ok:
+          topic: ${self:service}-${self:custom.stage}-critical-alerts-ok
+          notifications:
+          - protocol: https
+            endpoint: https://events.pagerduty.com/integration/.../enqueue
+        alarm:
+          topic: ${self:service}-${self:custom.stage}-critical-alerts-alarm
```
Comment (bulkan) [inline README.md]: should `${self.custom.stage}` be `${opt:stage}` ?

Comment (honzas) [reply README.md]: Yeah - you're right. The rest of readme.md uses opt:stage, so it should be the same - fixed now.

We use a bit different naming convetion - I missed that..

**Comment by Eric-Melb** [comment]
This is a great package and I'm looking forward to this feature. Thank you both.

**Comment by bulkan** [comment]
@honzas merged in via https://github.com/ACloudGuru/serverless-plugin-aws-alerts/commit/8b6b4b5ee5297bee2f6c03e10535edc0aaf9e4e3

**Pull request closed without merging**